draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -468,8 +474,8 @@
                 offerer MUST insert an SDP 'tls-id' attribute with the previously assigned value in the offer.
                 In addition, the offerer MUST insert an SDP 'setup' attribute, and one or more SDP 'fingerprint' attributes with
                 values that do not change the previously sent fingerprint set, in the offer. The value of the 'setup'
-                attribute SHOULD be set to 'actpass', in order to allow the answerer to establish a new DTLS
-                association with a different role, but MAY be set to the current negotiated role ('active' or
+                attribute MUST be set to either 'actpass', in order to allow the answerer to establish a new DTLS
+                association with a different role, or to the current negotiated role ('active' or

```

Comment:
I would prefer it to always be MUST 'actpass' and not allow currently negotiated role.  We can put another note that answerer is required to accept other setup attribute values in subsequent offers.